wdl/subworkflows/build_mappable_only_bed.wdl

```diff
@@ -0,0 +1,58 @@
+version 1.0
+
+
+import "../tasks/awk.wdl"
+import "../tasks/bedops.wdl"
+import "../tasks/hotspot1.wdl"
+import "../tasks/sed.wdl"
+
+
+workflow build_mappable_only_bed {
+    input {
+        BedopsSortBedParams sortbedparams
```

Comment:
sortbedparams -> params

Reply:
We should bind params here, not in input JSON. Something like `BedopsSortBedParams params = {}`

See e.g.: https://github.com/ENCODE-DCC/dnase-seq-pipeline/blob/98a5792859d0f92427d4c37ddc46328a8ebe9f1c/wdl/subworkflows/get_first_read_in_pair_from_bam.wdl#L12